app/js/trouble.js

```diff
@@ -72,52 +74,59 @@
 
 		function open() {
 			$dropdown.removeClass('no_transition');
-			$dropdown.data('status', 'open")'
+			$dropdown.data('status', 'open');
 			$dropdown.addClass('gbl_dropdown_active');
-			$dropdown.focus()
+			$dropdown.focus();
 			$toggle.addClass('gbl_dropdown_active');
 			$dropdown.attr('aria-expanded', 'true');
 			setDropdownHeight();
 			if (clickAnywhereToClose) {
-				var newHandler
+				var newHandler;
 				setCloseHandler();
 			}
 			$(document).trigger(ns + 'Open');
 		}
 
 		function toggleDropdown(e) {
-			e.preventDefault()
-			e.stopPropagation()
-			if ($dropdown.data('status') =='closed') {
-				function setStatus() {
-					newStatus = "closed"
+            
+			e.preventDefault();
+			e.stopPropagation();
+            
+            function setStatus() {
+				var newStatus = "closed";
+            }
+            
+            function getStatus() {
+				return
+				{
+				    status : "open";
 				}
+            }
+            
+			if ($dropdown.data('status') =='closed') {
+				setStatus();

```

Comment:
No need to call setStatus here. The removal of the function declarations from the conditional blocks was the key fix in this section of the code.